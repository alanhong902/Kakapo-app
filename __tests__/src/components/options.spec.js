import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { ImportOptions } from 'components/';
import { createComponentWithIntl } from '../helper';

test('<Options/> render', () => {
  const props = {};
  const tree = createComponentWithIntl(
    <MemoryRouter>
      <ImportOptions {...props} />
    </MemoryRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
