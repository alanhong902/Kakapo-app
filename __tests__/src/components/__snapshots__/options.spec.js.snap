// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Options/> render 1`] = `
<div
  className="downloads"
>
  <div
    className="media"
  >
    <h4
      className="add-sounds-title"
    >
      <span>
        Add a sound
      </span>
    </h4>
    <ul
      className="option-ul"
    >
      <li
        className="option options-kakapo"
      >
        <a
          href="/kakapo"
          onClick={[Function]}
        >
          <i
            className="icon-img-kakapo"
          />
          <span
            className="text"
          >
            <span>
              Kakapo favourites
            </span>
          </span>
        </a>
      </li>
      <li
        className="option options-youtube"
      >
        <a
          href="/youtube"
          onClick={[Function]}
        >
          <i
            className="icon-img-youtube"
          />
          <span
            className="text"
          >
            <span>
              YouTube
            </span>
          </span>
        </a>
      </li>
      <li
        className="option options-soundcloud"
      >
        <a
          href="/soundcloud"
          onClick={[Function]}
        >
          <i
            className="icon-img-soundcloud"
          />
          <span
            className="text"
          >
            <span>
              SoundCloud
            </span>
          </span>
        </a>
      </li>
      <li
        className="option options-custom"
      >
        <a
          href="/custom"
          onClick={[Function]}
        >
          <i
            className="icon-img-custom icon-add"
          />
          <span
            className="text"
          >
            <span>
              Custom link
            </span>
          </span>
        </a>
      </li>
    </ul>
    <div
      className="divider"
    />
    <ul
      className="option-ul"
    >
      <li
        className="option"
      >
        <a
          href="/playlist"
          onClick={[Function]}
        >
          <i
            className="icon-playlist"
          />
          <span
            className="text"
          >
            <span>
              Playlists
            </span>
          </span>
        </a>
      </li>
      <li
        className="option"
      >
        <a
          href="/settings"
          onClick={[Function]}
        >
          <i
            className="icon-settings"
          />
          <span
            className="text"
          >
            <span>
              Settings
            </span>
          </span>
        </a>
      </li>
    </ul>
  </div>
</div>
`;
